guides/common/modules/proc_migrating-a-host-to-use-pull-mode.adoc

```diff
@@ -0,0 +1,39 @@
+[id="migrating-a-host-to-use-pull-mode_{context}"]
+= Migrating a Host to Use Pull Mode
+
+To migrate a host from `katello-agent` to pull-based transport, you can use the `katello-pull-transport-migrate` utility.
+
+.Prerequisites
+* The host runs an Enterprise Linux derivative operating system.
+ifndef::satellite[]
+The utility should work on any system that uses Red Hat Subscription Manager, but it was tested only on Enterprise Linux.
+endif::[]
+* The host is registered to Katello.
+The utility will exit if the host is registered to Red Hat Subscription Manager directly.
+* You have enabled the {Team} {project-client-name} repository on the host.
+
+.Procedure
+. Install the `katello-pull-transport-migrate` package on the host.
+You can select one of the following options:
+** Add the package to a Content View that is assigned to the target host, promote the Content View, and then install the package.
+For more information, see {ContentManagementDocURL}Managing_Content_Views_content-management[Managing Content Views] and {ContentManagementDocURL}Installing_Packages_on_a_Host_content-management[Installing Packages on a Host] in _{ContentManagementDocTitle}_.
+** Execute a remote job on the target host to install the package.
+** Connect to the host using SSH and install the package manually:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# yum install katello-pull-transport-migrate
+----
```

Comment:
```suggestion
** Connect to the host using SSH and install `katello-pull-transport-migrate` manually.
```
This would be an alternative to an OS-specific example.